inspec/a2-api-integration/controls/iam_v2.rb

```diff
@@ -750,4 +750,285 @@
       expect(resp.parsed_response_body[:user_ids]).to eq([user])
     end
   end
+
+  describe "v2beta project rules API" do
+    before(:all) do
+       resp = automate_api_request("/apis/iam/v2beta/projects",
+        http_method: 'POST',
+        request_body: {
+          id: CUSTOM_PROJECT_ID,
+          name: "display name !#$#"
+        }.to_json
+      )
+      expect(resp.http_status).to eq 200
+    end
+
+    after(:all) do
+      resp = automate_api_request("/apis/iam/v2beta/projects/#{CUSTOM_PROJECT_ID}", http_method: 'DELETE')
+      expect(resp.http_status.to_s).to match(/200|404/)
+    end
+
+    context "when the project does not exist" do
+      before(:each) do
+        resp = automate_api_request("/apis/iam/v2beta/projects/#{CUSTOM_PROJECT_ID}", http_method: 'DELETE')
+        expect(resp.http_status.to_s).to match(/200|404/)
+      end
+
+      after(:all) do
+        resp = automate_api_request("/apis/iam/v2beta/projects",
+          http_method: 'POST',
+          request_body: {
+            id: CUSTOM_PROJECT_ID,
+            name: "display name !#$#"
+          }.to_json
+        )
+        expect(resp.http_status).to eq 200
+      end
+
+      describe "GET /iam/v2beta/projects/:id/rules" do
+        it "returns an empty list" do
```

Comment:
Can you make this test description a bit more explanatory? I was confused why three tests had the same name and had to read the setup detail (`project-not-found`) to sort it out.